lib/logging/CBasicLogConfigurator.cpp

```diff
@@ -145,7 +145,7 @@ EConsoleTextColor::EConsoleTextColor CBasicLogConfigurator::getConsoleColor(cons
 
 void CBasicLogConfigurator::deconfigure()
 {
-	auto l = CLogger::getGlobalLogger();
+	auto *l = CLogger::getGlobalLogger();
```

Comment:
```suggestion
	auto * l = CLogger::getGlobalLogger();
```

Reply:
all formatting is passed via clang-format.